src/steps/zsh.rs

```diff
@@ -194,7 +198,10 @@ pub fn run_oh_my_zsh(ctx: &ExecutionContext) -> Result<()> {
 
     ctx.run_type()
         .execute("zsh")
-        .env("ZSH", &oh_my_zsh)
```

Comment:
This line of code was introduced in [this commit](https://github.com/topgrade-rs/topgrade/commit/44b68a048f06febfb14fc592f3271f7800dcdfa3), and I haven't figured out why do we need this. IMHO, it is useless so I removed it.